packages/app/src/components/DocumentSidebar.module.css

```
@@ -0,0 +1,106 @@
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #333;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.15);
+}
+
+.toggleButton:hover {
+  background: #eee;
+}
+
+:global([data-color-scheme="dark"]) .sidebar {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)